packages/frontend/components/global/AppFooter/AppFooter.styles.ts

```diff
@@ -72,3 +81,26 @@ export const CopyrightText = styled.p`
   grid-area: copyright;
   padding-block-start: ${95 - 40}px;
 `;
+
+export const InstallationDesktop = styled.div`
+  align-items: center;
+
+  > p {
+    padding-block-end: ${pxToRem(3)};
+  }
+
+  ${respond(`display: none`, 70)}
```

Comment:
The missing `;` might be the reason why this component still shows on mobile:
<img width="514" alt="Screen Shot 2021-12-20 at 12 08 53 PM" src="https://user-images.githubusercontent.com/57107444/146826592-a7efe89e-8262-48f2-8b67-a545dc8ee03c.png">